### PR TITLE
Make init_tracing a macro to fix env! fallbacks

### PR DESCRIPTION
**Pull request by lbeschastny**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -12,10 +12,14 @@ pub use axum_tracing_opentelemetry::opentelemetry_tracing_layer;
 
 pub mod propagation;
 
-pub fn init_tracing(log_level: tracing::Level) {
+pub fn init_tracing_with_fallbacks(
+    log_level: tracing::Level,
+    service_name_fallback: &'static str,
```
Comment (raphamorim) [inline src/lib.rs:17]: Does need to be &'static?

File: src/lib.rs [hunk 2]
```diff
@@ -12,10 +12,14 @@ pub use axum_tracing_opentelemetry::opentelemetry_tracing_layer;
...
+    service_version_fallback: &'static str,
+) {
     let otel_rsrc = DetectResource::default()
-        .with_fallback_service_name(env!("CARGO_PKG_NAME"))
-        .with_fallback_service_version(env!("CARGO_PKG_VERSION"))
+        .with_fallback_service_name(service_name_fallback)
```
Comment (raphamorim) [inline src/lib.rs:21]: can we keep the variables name with the same pattern as the function? for example `fallback_service_name`

File: src/lib.rs (hunk 1 above)
Comment (lbeschastny) [reply src/lib.rs:17]: Yes:

```rust
pub fn with_fallback_service_name(self, fallback_service_name: &'static str) -> Self
pub fn with_fallback_service_version(self, fallback_service_version: &'static str) -> Self
```

File: src/lib.rs (hunk 2 above)
Comment (lbeschastny) [reply src/lib.rs]: https://github.com/nentgroup/telemetry-rust/pull/1/commits/c1b187577056727890b0bc73174fae9c7eb3cb60

**Comment by raphamorim** [comment]
Great, can include the README update as well in the PR? https://github.com/nentgroup/telemetry-rust/blob/main/README.md

**Review by raphamorim — APPROVED** [review]

**Pull request merged**